ruby/vampires.rb

```diff
@@ -1,11 +1,55 @@
-puts "what is your name?"
-name = gets.chomp
+puts "how many employees will be processed?"
+number = gets.chomp.to_i
 
-puts "how old are you? what you were you born?"
-age_year = gets.chomp
+survey = 0
+while survey < number
+
+  puts "what is your name?"
+  name = gets.chomp 
+    
+  puts "how old are you? what year were you born?"
+  age_year = gets.chomp.to_i
+  puts "should we order some garlic bread for you?"
+    garlic_bread = gets.chomp
+    
+  puts "would you like to enroll in the company's health insurance?"
+  health_insurance = gets.chomp
+  
+  puts "list out any allergies. type done when finished."
+  allergies = gets.chomp
+  
+    until allergies == "done"
+     puts "List out any allergies type done when finished."
+     allergies = gets.chomp
+    end
+  
+  if age_year < 100 && garlic_bread == "yes" || health_insurance == "yes"
```

Comment:
Consider checking  the birth year agains the Ruby time class `Time.now.year` this makes the code more dynamic and allows you the freedom of not having to rewrite the file with hardcoded values as years change.